index.bs

```diff
@@ -294,79 +325,76 @@ This callback will be invoked when there are changes to <i>target</i>'s intersec
     <dfn attribute for=IntersectionObserver>\[[QueuedEntries]]</dfn> and
     <dfn attribute for=IntersectionObserver>\[[ObservationTargets]]</dfn> slots,
     which are initialized to empty lists and internal
+    <dfn attribute for=IntersectionObserver>\[[callback]]</dfn>,
     <dfn attribute for=IntersectionObserver>\[[root]]</dfn>,
     <dfn attribute for=IntersectionObserver>\[[rootMargin]]</dfn> and
     <dfn attribute for=IntersectionObserver>\[[threshold]]</dfn> slots, which
-    are initialized to the corresponding attributes of the {{options}} parameter
-    passed to <a>IntersectionObserver(callback, options)</a>.
+    are initialized by <a>IntersectionObserver(callback, options)</a>.
 
 <h3 id='3-2-algorithms'>Algorithms</h2>
```

Comment:
All of these algorithms are a bit too implicit on what they are operating. I would state them as e.g. "To **notify intersection observers**, given a unit of related similar-origin browser contexts _unit_, run these steps: ... If _unit_'s internal  [[IntersectionObserverTaskQueued]] slot is..."